### PR TITLE
Still more changelog entries.

### PR DESCRIPTION
**Pull request by kevingranade**
Under 19,000 commits left to process.

### CODE REVIEW

File: data/changelog.txt [hunk 1]
```diff
@@ -27,10 +28,15 @@ Ranch and Ranch-related missions.
 Monster grab and pull attacks.
 Recipes based on offal and corpse waste.
 Cheesemaking.
-Tanks Mod
 biodiesel
 Alchohol stoves.
 Vehicle based tools, street sweepers, tractors, plows, planters.
+Seasonal veriation in foliage.
```
Comment (Night-Pryanik) [inline data/changelog.txt]: Variation.

File: data/changelog.txt [hunk 2]
```diff
@@ -44,11 +50,19 @@ Lots of dialogs allow use or consumption of items from immediate surroundings as
 Enhancements to AIM.
 Improved explosion animation.
 Ambient sound effects.
+Added vehicle direction indicator in tiles mode.
+Prevented spam about player being tired.
+Removed inability for vehicles to drive over fungal beds.
+Fixed a number of issues around monster spawning: Wraith, ants spawning in sight of layer, animals spawning underground.
```
Comment (Leland) [inline data/changelog.txt]: > ants spawning in sight of layer

Is that meant to be *p*layer?

File: data/changelog.txt (hunk 1 above)
Comment (kevingranade) [reply data/changelog.txt]: Thank you.

File: data/changelog.txt (hunk 2 above)
Comment (kevingranade) [reply data/changelog.txt]: It is, thanks for proofing.

**Pull request merged**